moto/shield/models.py

```diff
@@ -0,0 +1,202 @@
+"""ShieldBackend class with methods for supported APIs."""
+
+from typing import Any, Dict, List
+
+from moto.core.base_backend import BackendDict, BaseBackend
+from moto.core.common_models import BaseModel
+from moto.moto_api._internal import mock_random
+from moto.shield.exceptions import (
+    InvalidParameterException,
+    InvalidResourceException,
+    ResourceAlreadyExistsException,
+    ResourceNotFoundException,
+    ValidationException,
+)
+from moto.utilities.tagging_service import TaggingService
+
+
+class Protection(BaseModel):
+    def __init__(
+        self, account_id: str, name: str, resource_arn: str, tags: List[Dict[str, str]]
+    ):
+        self.name = name
+        self.resource_arn = resource_arn
+        self.protection_id = str(mock_random.uuid4())
+        self.tags = tags
```

Comment:
Nitpick - it looks like tags is never used, so we don't have to store it here

Reply:
@bblommers  Removed the tags